index.js

```diff
@@ -57,7 +60,10 @@ var defaults = {
   'text-size': 16,
   'icon-opacity': 1,
   'icon-rotate': 0,
-  'icon-size': 1
+  'icon-size': 1,
+  'circle-radius' : 5,
```

Comment:
You have to remove `circle-radius` from the defaults. Otherwise all points will be rendered with a circle, even if no `circle-radius` is in the style object.